muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/communicator/body/application/messages.tsx

```diff
@@ -409,6 +393,13 @@ class CommunicatorMessages extends BodyScrollLoader<
 
     return (
       <BodyScrollKeeper hidden={!!this.props.currentThread}>
+        {this.props.messages.location === "trash" ? (
```

Comment:
Tän vois tehdä lyhyemmin ja ilman else-käsittelyä ihan vaan :

`{this.props.messages.location === "trash" && ...}`

Ei merkittävä asia sinänsä.

Reply:
Muutettu